modules/simple-payments/simple-payments.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+class Jetpack_Simple_Payments {
+	// These have to be under 20 chars because that is CPT limit.
+	static $post_type_order = 'jp_pay_order';
+	static $post_type_product = 'jp_pay_product';
+
+	static $shortcode = 'simple-payment';
```

Comment:
There is precedence for using the product name even if it's a plural form e.g. `[8tracks ...]` or `[googleapps ...]` for shortcodes available on wpcom https://en.support.wordpress.com/shortcodes/ 
But those are highly branded services that we can't change.

I can't find anything specific around best practices here https://developer.wordpress.org/plugins/shortcodes/#shortcode-best-practices
There is the prefix thing so something like `[jetpack-simple-payments ]`  would make sense to use the full product name. Not saying we need the prefix however.

I do find it more natural to use the singular form but the plural form is more correct. That said I suggest we go with `[simple-payment]` since it's more user friendly.